lib/ansible/module_utils/yumdnf.py

```diff
@@ -126,15 +126,25 @@ def __init__(self, module):
         # default isn't a bad idea
         self.lockfile = '/var/run/yum.pid'
 
+    def is_lockfile_pid_valid(self):
```

Comment:
This is an abstract base class so use @abstractmehod instead of raising NotImplementedError.

Reply:
TIL, thanks!